x-pack/plugins/actions/server/types.ts

```diff
@@ -173,3 +175,10 @@ export interface ResponseSettings {
 export interface SSLSettings {
   verificationMode?: 'none' | 'certificate' | 'full';
 }
+
+export interface ConnectorToken extends SavedObjectAttributes {
+  connectorId: string;
+  tokenType: string;
+  token: string;
+  expiresIn: string;
```

Comment:
`expiresIn` indicates the amount of time from a start time at which the token will expire. Do you think `expiresAt` would be a more accurate name?